data/wordlists.yml

```diff
@@ -5630,7 +5630,7 @@ pron:
   categories:
   - misc
 
-[COMBO] NEW:
+NEW:
```

Comment:
Too generic of a wordlist name. Should include either `Weakpass` or `COMBO` to indicate the specific wordlist.